src/applications/accredited-representative-portal/actions/user.js

```diff
@@ -1,6 +1,3 @@
-import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/api';
```

Comment:
this `user.js` stuff is for local dev to avoid login. We should keep it out of PRs.

Reply:
removing the user file breaks the build, looks like this is being used in a few other places

Reply:
nvm, adding it back in after our convo, thank you!